### PR TITLE
(Updated) Add Granblue Fantasy Wiki presence

### PR DESCRIPTION
**Pull request by kyrie25**
A presence for https://gbf.wiki/
### Updated from the previous pull request, these presences are available while undergoing these tasks:
  * Logging in  
  * Creating an account    
  * Making a collection tracker  
  * Searching for:  
  * Editing preferences  
  * Viewing watchlist  
  * Looking up contributions  
  * Viewing revision history  
  * Editing  
  * Viewing main page
  * Viewing pages
### Screenshots:
image
image
image
image
image
image
image
image

### CODE REVIEW

File: websites/G/Granblue Fantasy Wiki/presence.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+const presence = new Presence({
+    clientId: "914354609370329098"
+  }),
+  browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (EncryptedDev) [inline websites/G/Granblue Fantasy Wiki/presence.ts:4]:
```suggestion
  browsingTimestamp = Math.floor(Date.now() / 1000);

```

File: websites/G/Granblue Fantasy Wiki/presence.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
+const presence = new Presence({
+    clientId: "914354609370329098"
+  }),
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "vyrnball"
+  };
+  presenceData.startTimestamp = browsingStamp;
```
Comment (EncryptedDev) [inline websites/G/Granblue Fantasy Wiki/presence.ts:10]: Set this in the PresenceData object

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/G/Granblue Fantasy Wiki/presence.ts (hunk 2 above)
Comment (kyrie25) [reply websites/G/Granblue Fantasy Wiki/presence.ts]: Fixed

File: websites/G/Granblue Fantasy Wiki/presence.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    clientId: "914354609370329098"
+  }),
+  browsingTimestamp = Math.floor(Date.now() / 1000);
+
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "vyrnball"
+  };
+  presenceData.startTimestamp = browsingTimestamp;
```
Comment (EncryptedDev) [inline websites/G/Granblue Fantasy Wiki/presence.ts:11]: Move this into the PresenceData object above

File: websites/G/Granblue Fantasy Wiki/presence.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  }
+  else if (document.location.href.indexOf("history") > -1) {
+    presenceData.details = "Viewing revision history of:";
+    presenceData.state = document.querySelector(".firstHeading").textContent;
+  }
+  else if (document.location.href.indexOf("edit") > -1) {
+    presenceData.details = "Editing:";
+    presenceData.state = document.querySelector(".firstHeading").textContent;
+  }
+  else if (document.querySelector(".firstHeading") !== null) {
```
Comment (EncryptedDev) [inline websites/G/Granblue Fantasy Wiki/presence.ts:44]:
```suggestion
  else if (document.querySelector(".firstHeading")) {
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/G/Granblue Fantasy Wiki/presence.ts (hunk 3 above)
Comment (kyrie25) [reply websites/G/Granblue Fantasy Wiki/presence.ts]: Fixed

**Comment by kyrie25** [comment]
Minor update:
- Added presence for: `Viewing the character tier list`
- Added small image for: `Viewing the character tier list`, `Logging in`, `Creating an account`, `Searching for:`, `Editing`, `Viewing page:`
image
image

**Review by Slowlife01 — APPROVED** [review]

File: websites/G/Granblue Fantasy Wiki/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  "logo": "https://i.imgur.com/k5QiAPC.png",
+  "thumbnail": "https://i.imgur.com/3vIpCD7.png",
+  "color": "#2F4F7A",
+  "tags": [
+    "granblue",
+    "fantasy",
+    "gbf",
+    "wiki"
+  ],
+  "category": "anime"
```
Comment (Slowlife01) [inline websites/G/Granblue Fantasy Wiki/dist/metadata.json:22]: Hmm anime?

**Review by Slowlife01 — CHANGES_REQUESTED** [review]

File: websites/G/Granblue Fantasy Wiki/dist/metadata.json (hunk 5 above)
Comment (kyrie25) [reply websites/G/Granblue Fantasy Wiki/dist/metadata.json:22]: Considering I based this of from Bas950's _Azur Lane Wiki_ and he wrote it down as "anime",and that both games shares the same genre.  
So yeah, anime

Comment (kyrie25) [reply websites/G/Granblue Fantasy Wiki/dist/metadata.json:22]: @Slowlife01 what category do you think is better for this presence and I'll fix it ASAP

Comment (kyrie25) [reply websites/G/Granblue Fantasy Wiki/dist/metadata.json]: Changed it to `game`, minor mistake sorry

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**